Resources/js/translator.js

```diff
@@ -149,6 +149,11 @@
             );
 
             var _number  = parseInt(number, 10);
+            parameters = parameters || {};
+
+            if (!parameters.count) {
```

Comment:
this is wrong. You need to check whether it is defined or no. ``0`` should not be replaced if it is passed explicitly.